Content.Server/Shuttles/Commands/FTLDiskBurnerCommand.cs

```diff
@@ -0,0 +1,122 @@
+using Content.Server.Administration;
+using Content.Server.Labels;
+using Content.Shared.Administration;
+using Content.Shared.Shuttles.Components;
+using Robust.Shared.Console;
+using Robust.Shared.Map.Components;
+using Robust.Shared.Prototypes;
+
+
+
+
+namespace Content.Server.Shuttles.Commands;
+
+/// <summary>
+/// Creates FTL disks, to maps, grids, or entities.
+/// </summary>
+[AdminCommand(AdminFlags.Fun)]
+
+public sealed class FTLDiskBurnerCommand : IConsoleCommand
+{
+    [Dependency] private readonly IEntityManager _entManager = default!;
+    [Dependency] private readonly IEntitySystemManager _entSystemManager = default!;
+
+    public string Command => "FTLdiskburner";
+    public string Description => Loc.GetString("ftl-disk-burner-desc");
+    public string Help => Loc.GetString("cmd-ftl-disk-burner-help");
+
+    [ValidatePrototypeId<EntityPrototype>]
+    public const string CoordinatesDisk = "CoordinatesDisk";
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (args.Length == 0)
+        {
+            shell.WriteError("Not enough arguments.");
+            return;
+        }
+
+
+        var player = shell.Player;
+
+        if (player == null)
+        {
+            shell.WriteLine("Only a player can run this command.");
+            return;
+        }
+
+        if (player.AttachedEntity == null)
+        {
+            shell.WriteLine("You don't have an entity to spawn a disk at.");
+            return;
+        }
+
+        EntityUid entity = player.AttachedEntity.Value;
+        var coords = _entManager.GetComponent<TransformComponent>(entity).Coordinates;
+
+        foreach (var destinations in args)
+        {
+            if (destinations == null)
+                return;
+
+            /// make sure destination is an id.
+            EntityUid dest;
+            if (_entManager.TryParseNetEntity(destinations, out var nullableDest))
+            {
+                if (nullableDest == null)
+                    continue;
```

Comment:
You're right it shouldn't ever get hit since it _should_ return telling you there's not enough arguments. But just in case I didn't think of something like trying to fly to nullspace I went ahead and did both. Either way it shouldn't die if you somehow manage to do that on live.